app/models/tournament.rb

```diff
@@ -0,0 +1,3 @@
+class Tournament < ApplicationRecord
+  belongs_to :booking
```

Comment:
i think it should be:

```suggestion
  has_many :bookings
```